server/graphql-schema.js

```diff
@@ -840,6 +865,31 @@ const graphqlSchema = gql`
     failedReason: AssertionFailedReason
   }
 
+  enum TestPlanVersionPhase {
```

Comment:
Looks like this `enum` is getting duplicated

Reply:
Addressed with [this commit](https://github.com/w3c/aria-at-app/pull/1205/commits/7d8304abf55f2c01ff5aaf3fb98a9fb902b1e518)